docs/source/reference/rest_api.rst

```diff
@@ -0,0 +1,35 @@
+.. _rest_api:
+
+REST API Usage 
+======================
+
+We can access EvaDB through REST APIs using two endpoints shown below:
```

Comment:
Could you add documentation on how to start the REST server?

Reply:
yes. I've added documentation to start the server and moved it below python API